service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationAzureJobFlight.java

```diff
@@ -60,7 +60,9 @@ public RunImputationAzureJobFlight(FlightMap inputParameters, Object beanBag) {
 
     addStep(
         new PrepareImputationInputsStep(
-            flightBeanBag.getPipelinesService(), flightBeanBag.getImputationConfiguration()),
+            flightBeanBag.getPipelinesService(),
```

Comment:
note that because the Azure flight also uses this step, for now if we ran the Azure flight and it failed, we'd get multiple failed run counts if the error happened at this step or later. I'm ok leaving this as is for now and its being resolved when we implement TSPS-181

Reply:
we could move this to be the first step of the flight and remove the counter from CheckLeonardoHealthStep if we wanted to mitigate it in the meantime.  I dont think this has dependencies on any of the preceding steps and it would make it easier to clean up when we do TSPS-181

Reply:
great idea, will do